### PR TITLE
fixed the video height issue at mobile device

### LINKED ISSUE #7980: [Docs] Video height issue at mobile devices
#### Current State
when we visit  there are some issue at mobile devices in media queries we have to change height of vedio because after start the video most of the spaces did not in use , so i am showing some pictures current and desired behaviour , first 2 pictures are current behavior in mobile devices and last two are expected behaviour in mobile devices

#### Desired State

image
image
image
image

--
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Debjyoti0404**
**Notes for Reviewers**
Fixed the height issue for mobile screen and removed unwanted black space which was appearing while playing the video.
This PR fixes #7980 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 19d2d1ae3a180a32b5dc0ea4ffb8e5073c1dddc3 at: https://64c5612d457ce716e5868ba6--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8b18b0ca5b61c511634713fde5360b6f17c0a90f at: https://64c6d2499e67f33c547adae2--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8b18b0ca5b61c511634713fde5360b6f17c0a90f at: https://64c6d2559e67f33bbe7adb07--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1cfcd90848beecbfc72f94c934896e3bfc166e94 at: https://64c6d33e457ce73ed7868ee2--meshery-docs-preview.netlify.app

File: docs/Gemfile.lock
Comment (thisiskaransgit) [inline docs/Gemfile.lock:1]: @Debjyoti0404, Revert this change and please refrain from pushing any dependency changes.

**Comment by l5io** [comment]
🚀 Preview for commit 1cfcd90848beecbfc72f94c934896e3bfc166e94 at: https://64c7a660c7c66f097d9b47f9--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3c1e79d82ac2e906ec62450b6ff545e831dd7be4 at: https://64c7b5e1eb77ac179f5948cb--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0a672d72250197a25bcb64cf8787098492984e14 at: https://64c7b758dcd8ed14df5c25c7--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2e56259cf969b451d1dba6c2c8bbe217fe04ca23 at: https://64c7ba4966879f0071af5889--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@Debjyoti0404 is this PR redundant given the recently merged PR?

**Comment by Debjyoti0404** [comment]
> @Debjyoti0404 is this PR redundant given the recently merged PR?

Yes, the css part is redundant. I updated the PR.

**Comment by l5io** [comment]
🚀 Preview for commit f87442899d5e16eae680b79d72b83806a0fb8722 at: https://64c87a287dd4292f57eacd6e--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f87442899d5e16eae680b79d72b83806a0fb8722 at: https://64c87a3cad29fb31f2cbc6d6--meshery-docs-preview.netlify.app

**Comment by thisiskaransgit** [comment]
closed as completed via #8322

**Comment by Debjyoti0404** [comment]
> closed as completed via #8322

Hey @thisiskaransgit, we still need to update [this lines](https://github.com/meshery/meshery/pull/8313/files) as the class is not added in the html tag.
Otherwise it is still as previous state with all the black space in the video section.
image
